components/healthcare/code/healthcare-bbe/why-ballerina-for-healthcare-apps.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: 'Why is Ballerina the way to write healthcare applications?'
-description: 'Java and HAPI have long been the de facto for health app development. However, cloud native healthcare application development goes beyond dealing with objects and frameworks. This domain needs a fresh architecture and technology that treats concepts like FHIR and HL7 as first class citizens in the language, alongside concepts like JSON and APIs.<br/><br/>
+title: 'What makes Ballerina the ideal choice for developing healthcare applications?'
```

Comment:
Can you update the lines 50 * 58 in https://github.com/ballerina-platform/ballerina-dev-website/blob/3402ba93f936699845fe4fee839b11741308b612/components/healthcare/code/Code.js#L50 as the new title does not match the permalink.

Reply:
Done.